src/frontend/utils/formatering.ts

```diff
@@ -1,5 +1,16 @@
+import { parseISO } from 'date-fns';
+
 import { Adresse } from '../typer/person';
 
+export const datoFormat = {
+    day: '2-digit',
+    month: '2-digit',
+    year: 'numeric',
+} as const;
+
 export const formaterAdresse = (adresse: Adresse) => {
     return `${adresse.adresse}, ${adresse.postnummer} ${adresse.poststed}`;
 };
+export const formaterIsoDato = (dato: string): string => {
+    return parseISO(dato).toLocaleDateString('no-NO', datoFormat);
```

Comment:
Et alternativ er å bruke format med 
```suggestion
    return format(parseISO(date), "dd.MM.yyyy");
```

Reply:
Er det noe forskjell?

Reply:
Nei egentlige ikke men då trenger man ikke `datoFormat`  og tenker ` "dd.MM.yyyy"` er tydelig nok 👀